### PR TITLE
Prometheus: Fix queries with label_replace remove the $1 match when loading query editor

### LINKED ISSUE #18455: Prometheus: Queries with label_replace remove the $1 match when loading query editor (dashboard & explore)
**What happened**:
When clicking on the "explore" link from a panel I already have (which contains a label_replace query), the explore view is correctly opened but the query remove the "$1" character in the args of label_replace function.
It makes the query impossible to copy/paste to the prometheus UI.

what is displayed in Explore UI:
```
label_replace(http_request_seconds),"pod", "", "pod_name","(.*)")
```

**What you expected to happen**:
The prometheus query should keep displaying the $1 character.
what should be displayed in Explore UI:
```
label_replace(http_request_seconds),"pod", "$1", "pod_name","(.*)")
```

**Environment**:
- Grafana version: all from 6.x to 6.3.2
- Data source type & version: prometheus v2.11
- OS Grafana is installed on: official docker image
- User OS & Browser: Mac OS High Sierra, Safari 12.1.2
- Grafana plugins: None

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:
After an internal discussion with @davkal we've decided to remove `PlaceholdersBuffer` which fixes #18455 

**Which issue(s) this PR fixes**:
Fixes: #18455

**Special notes for your reviewer**:
Make sure anything else didn't break in QueryFields with this change. I've tested Tabbing and suggestions.

### CODE REVIEW

File: public/app/features/explore/QueryField.tsx [hunk 1]
```diff
@@ -325,11 +319,7 @@ export class QueryField extends React.PureComponent<QueryFieldProps, QueryFieldS
 
       const insertTextOperation = nextChange.operations.find((operation: any) => operation.type === 'insert_text');
       if (insertTextOperation) {
-        const suggestionText = insertTextOperation.text;
-        this.placeholdersBuffer.setNextPlaceholderValue(suggestionText);
-        if (this.placeholdersBuffer.hasPlaceholders()) {
-          nextChange.move(this.placeholdersBuffer.getNextMoveOffset()).focus();
-        }
+        return undefined;
```
Comment (hugohaggmark) [inline public/app/features/explore/QueryField.tsx]: Not sure what impact removing the `.focus();` part has.

File: public/app/features/explore/QueryField.tsx [hunk 2]
```diff
@@ -136,9 +132,7 @@ export class QueryField extends React.PureComponent<QueryFieldProps, QueryFieldS
         .change()
         .insertText(' ')
         .deleteBackward();
-      if (this.placeholdersBuffer.hasPlaceholders()) {
-        change.move(this.placeholdersBuffer.getNextMoveOffset()).focus();
-      }
+
```
Comment (hugohaggmark) [inline public/app/features/explore/QueryField.tsx]: Not sure what impact removing the `.focus();` part has.

**Review by davkal — APPROVED** [review]
LGTM

**Comment by torkelo** [comment]
added missing cherry-pick needed label :)

**Pull request merged**